explainability-core/src/main/java/org/kie/trustyai/explainability/model/dataframe/DataframeMetadata.java

```diff
@@ -98,7 +123,7 @@ public synchronized void add(Output output) {
     private void updateCachedColumnNames() {
         List<String> newNames = new ArrayList<>();
         for (int i = 0; i < this.names.size(); i++) {
-            if (this.nameAliases.get(i) != null) {
+            if (!this.nameAliases.get(i).equals("")) {
```

Comment:
To avoid potential NPE
```suggestion
            if (!"".equals(this.nameAliases.get(i)) {
```

Reply:
done